### PR TITLE
BL 209 - FE as an admin i can visualize feedback from various user groups to get a global picture of how people are feeling

### PR DESCRIPTION
**Pull request by albert-szabo**
## Description

We created one wireframe and two Figma mocks to show how administrators will be able to review feedback on mentors submitted by mentees. The wireframe included stars on both views with a three-star system to match the DS sentiment analysis that was planned to return one of three words: "negative" being represented by one star, "neutral" being represented by two stars, and "positive" being represented by three stars.

Initial versions of the two Figma mocks based on the wireframe were presented to UX manager Swapnali. Some accessibility-minded tweaks were made based on her feedback, and she approved them generally.

The revamped Figma mocks were presented at the stakeholder meeting on 05 OCT 2022. Our stakeholder Jason and DS manager Robert provided feedback advising the removal of the stars and replacement of them with simple text. Jason also suggested alternating distinctive background colors on the rows to more easily distinguish one block of text from another.

These suggested improvements were implemented in Figma, resulting in the final mocks, screenshots of which are included to be merged.

#### Note on Viewing the PNG Files

Reviewers may have to open the second (from the top) commit individually under the "commits" tab to see the PNG files rendered correctly. Under the "files changed" tab, they may not display since GitHub tries to render them as code blocks.

### Figma Link

[Figma File](https://www.figma.com/file/cwOoFddP7i0BdcP1oB7Kco/Underdog-Devs-Ant-Design?node-id=92%3A0)

### Video Link

[Loom Video](https://www.loom.com/share/755df391623a463e8bf3b4670ed8577b)

### Ticket Link

[Jira Ticket (BL-209)](https://bloomtechlabs.atlassian.net/browse/BL-209?atlOrigin=eyJpIjoiNjVmMmUzYmI5NDE0NDgwZGFiMzIxNzcxYjIyNDcxZjUiLCJwIjoiaiJ9)

#### Assignees 

The team members who worked on this ticket were @albert-szabo, @UlisesGatica, and Nelson Maldonado.

## Checklist:

- [x] My design/diagram follows the style guidelines of this project
- [x] My docs are organized within an appropriate folder structure in our repo
- [x] Pull request description clearly describes docs made & motivations for said additions

### CODE REVIEW

**Review by randynobx — APPROVED** [review]
These mockups look good. Clean design, no clutter.

The removal of the stars in favor of text and the banded coloring of the rows is a huge improvement IMHO.

**Review by zoebrantley — APPROVED** [review]
README is descriptive and walks us through the process of how the mocks have changed well. The use of negative, neutral, and positive really help to make the table to look extra clean/professional. Great work.

**Review by JeffDesch — APPROVED** [review]
I really like the clean look of the newest design. The middle field as text instead of star rating gives DS some more flexibility in implementing various forms of sentiment analysis.

**Review by TophertheCoder — APPROVED** [review]
Obviously this is all dummy data at the moment but when it actually starts being used, the clear boundaries of text fields, rating system, and labeling will make it very clear and understandable to use, edit, and read. Great job all around!

**Review by tim-sansone — APPROVED** [review]
Overall this looks good. The display is clean and easy to read/understand at a glance. The suggested interface also seems to be straightforward (click on a specific mentor to see the specific feedback for that mentor).

One question: is the "result" column on the Mentor-Feedback page some sort of collective average of all their feedback? If so, I think labeling the column something to the effect of "average rating" would be more clear for the admin using this page.

Similar question for the modal: where does the content of the "result" column come from? I think labeling that column something like "overall rating" or "overall feedback" would communicate more clearly what the information in that column is supposed to represent.

Is this ticket supposed to include the FE implementation (the React components) of these designs as well? Will there be a separate pull request on the FE with this same ticket#?

**Review by matt-arofin — APPROVED** [review]
Love the way the ReadMe walks through all the work that went into producing the final mocks! A thoroughly documented process and attention to detail that more than justifies the high quality of the end results

**Comment by albert-szabo** [comment]
To touch on the feedback that @tim-sansone gave, I agree that more descriptive names in both tables for the feedback aggregation columns would be a good future improvement. We did have better column names above our stars, but those were removed and a more generic name was used when the simple text was added.

Future teammates may have to circle back with the DS team on whether the sentiment analysis has been changed to return more than one simple word.

Because this feature is being shelved, we're just committing the design / UX work for now and not implementing the FE components at this time, so there is no forthcoming FE pull request for those components.

**Review by ashurik522 — APPROVED** [review]
Great clean design, well done!

File: figma-screenshots/AdminFeedbackViews/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<img title="Mentor-Modal" alt="A Figma mock with a table that has four rows and three columns" src="./Mentor-Modal.png">
+
+On the initial wireframe and Figma mocks, stars were included in the form of a three-star system with orange stars to approximately match the color of the UD logo. The plan was to use the filling of those stars to represent the overall sentiment analysis returned from DS as one of three words. "Negative" would be represented as one filled-in orange star out of three, "neutral" as two, and "positive" as all three filled. In response to negative feedback from both stakeholder Jason and DS manager Robert at the stakeholder meeting on 05 OCT 2022, the middle column of stars was removed and replaced with simple text. Therefore, stars are no longer present in the final Figma mocks.
+
+In addition, stakeholder Jason's feedback suggesting alternating background colors on the rows was implemented to allow the user to clearly delineate where one row (and especially block of text) ends and another begins.
```
Comment (joanterm) [inline figma-screenshots/AdminFeedbackViews/README.md:19]: I like the design and adding different colors for each row was definitely a good idea.

Comment (roxannebobby) [reply figma-screenshots/AdminFeedbackViews/README.md:19]: I also like the different colors. Well done!

**Review by roxannebobby — APPROVED** [review]
Great work, the wireframes are clear and uncluttered and easy to see what info you need.

**Pull request merged**